include/pocl_context.h

```diff
@@ -43,6 +43,7 @@ struct pocl_context32 {
   uint printf_buffer_capacity;
   uint global_var_buffer;
   uint work_dim;
+  uint unreachable;
```

Comment:
Please add a comment as it's not obvious to outsiders what this is and how it is used.

Reply:
renamed the flag & added a comment